sql-cli/sql_cli/configuration.py

```diff
@@ -102,32 +88,3 @@ def write_value_to_yaml(self, section: str, key: str, value: str, filepath: Path
 
         with filepath.open(mode="w") as fp:
             yaml.dump(yaml_config, fp)
-
-    def write_config_to_yaml(self) -> None:
```

Comment:
We're removing this as I believe we should not be doing this as it has side effects. E.g. When the config object is loaded from YAML, with loading the YAML with expand the env vars to replace the placeholders with their values. Now, in our existing implementation, we call this method to write such config instance back to the YAML which has a side effect of replacing the env var placeholders with the actual values that the Config instance holds.